src/components/Card.js

```diff
@@ -23,27 +29,34 @@ export default class Card {
     );
 
     this._likeButton.addEventListener("click", () => {
-      this._handleLikeIcon();
+      this.handleLikeIcon();
     });
 
     this._cardElement
       .querySelector(".elements__trash-button")
       .addEventListener("click", () => {
-        this._handleTrashCard();
+        this._handleDeleteButton(this);
       });
 
     this._cardImage.addEventListener("click", () => {
       this._handleImageClick(this._cardImage);
     });
   }
 
-  _handleLikeIcon() {
+  handleLikeIcon() {
```

Comment:
Needs correction: Please remove this function. You should declare the setter for the likes and use it after updating likes in index.js, so the likes will be equal to the server likes and `isLiked` flag will be actual.

```js
setIsLiked(isLiked) {
  this._isLiked = isLiked;
  this.updateLiked();
}
```